src/backend/distributed/commands/trigger.c

```diff
@@ -727,6 +729,49 @@ ErrorOutForTriggerIfNotSupported(Oid relationId)
 }
 
 
+/*
+ * ErrorIfRelationHasUnsupportedTrigger throws an error if given relation has
+ * a trigger that is not supported by Citus.
+ */
+void
+ErrorIfRelationHasUnsupportedTrigger(Oid relationId)
+{
+	List *relationTriggerList = GetExplicitTriggerIdList(relationId);
+
+	Oid triggerId = InvalidOid;
+	foreach_oid(triggerId, relationTriggerList)
+	{
+		ObjectAddress triggerObjectAddress = InvalidObjectAddress;
+		ObjectAddressSet(triggerObjectAddress, TriggerRelationId, triggerId);
+
+		/* triggers that depend on extensions are not supported */
+		if (ObjectAddressDependsOnExtension(&triggerObjectAddress))
+		{
+			ErrorOutForTriggerDependsOnExtension(GetTriggerNameById(triggerId));
+		}
+	}
+}
+
+
+/*
+ * ErrorOutForTriggerDependsOnExtension throws an error for given trigger that
+ * that presumably depends on an extension.
+ *
+ * Moved into a function to make the error message consistent.
+ */
+static void
+ErrorOutForTriggerDependsOnExtension(char *triggerName)
+{
+	ereport(ERROR, (errmsg("trigger %s depends on an extension and this "
```

Comment:
Errors with identifiers are harder to read without quotes. Original location had quotes as well, caused unnecessary, changes in tests.

```suggestion
	ereport(ERROR, (errmsg("trigger \"%s\" depends on an extension and this "
```